packages/example-getting-started/docs/1-prerequisites-and-setup.md

```diff
@@ -9,26 +9,44 @@ certain technologies, languages and concepts.
 - [npm](https://www.npmjs.com/)
 - [REST](https://en.wikipedia.org/wiki/Representational_state_transfer)
 
-## Setup
-1. Install the new loopback CLI toolkit.
+
+Lastly, you'll need to install the LoopBack 4 CLI toolkit:
 ```
 npm i -g @loopback/cli
 ```
-2. Download the "getting-started" application.
+
+If you'd like to continue the tutorial, then jump to the
+[next section here](2-scaffold-app.md).
+
+If you'd like to see the final results of this tutorial as an example
+application, then follow these steps:
+
+1. Run the `lb4 example` command to clone the getting-started repository:
```

Comment:
I'd say `Run the 'lb4 example' command to select and clone the getting-started repository` to be clear